VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+Even though there is no obligation for validators to self-bond Atoms, delegators should want their validator to have self-bonded Atoms in their staking pool. In other words, validators should have skin-in-the-game. 
+
+In order for delegators to have some guarantee about how much skin-in-the-game their validator has, the latter can signal a minimum amount of self-bonded Atoms. If a validator's self-bond goes below the limit that it predefined, this validator and all of its delegators will unbond. 
+
+### How to prevent concentration of stake in the hands of a few top validators?
+
+For now the community is expected to behave in a smart and self-preserving way. When a mining pool in Bitcoin gets too much mining power the community usually stops contributing to that pool. Cosmos will rely on the same effect initially. In the furture, other mechanisms will be deployed to smoothen this process as much as possible:
+
+* **Penalty-free re-delegation:** This is to allow delegators to easily switch from one validator to the other, in order to reduce validator stickiness 
+* **Hack bounty:** This is an incentive for the community to hack validators. There will be bounties proportionate to the size of the validator, so that a validator becomes bigger of a target as its stake grows.
+* **UI warning:** Users will be warned by the wallet’s UI if they want to delegate to a validator that already has a significant amount of staking power
+
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+It is highly probable that they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block provisions and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called `photons`.
+
+
+## Technical requirements 
+
+### What are hardware requirements?
+
+Validators should expect to provision one or more data center locations with redundant power, networking, firewalls, HSMs and servers.
+We expect a modest level of hardware specifications will be needed initially and requirements to rise as network use rises. Participating in the testnet is the best way to learn more.
+
+### What are software requirements?
+
+In addition to running a cosmos node, validators should develop monitoring, alerting and management solutions.
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating Denial of Service attacks and syncing new nodes on the network.
```

Comment:
DDos

Reply:
The syncing part is not reduced. Let's remove the last part of the sentence.

Reply:
ok